Mathlib/Topology/Sets/Closeds.lean

```diff
@@ -430,6 +433,68 @@ order isomorphism. -/
 def orderIsoSubtype' : IrreducibleCloseds α ≃o { x : Set α // IsClosed x ∧ IsIrreducible x } :=
   equivSubtype'.toOrderIso (fun _ _ h ↦ h) (fun _ _ h ↦ h)
 
+/-! ### Partial order structure on irreducible closed sets and maps thereof-/
+
+instance : PartialOrder (IrreducibleCloseds α) := inferInstance
+
+/-- The map on irreducible closed sets induced by a continuous map `f`. -/
+def map {f : β → α} (hf : Continuous f)
+    (c : IrreducibleCloseds β) : IrreducibleCloseds α where
+  carrier := closure (f '' c.carrier)
+  isIrreducible' := c.isIrreducible.image f hf.continuousOn |>.closure
+  isClosed' := isClosed_closure
+
```

Comment:
Can you add a lemma saying `(map hf s : Set _) = closure (f '' s)`, which should be called `coe_map`? Thanks.